src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class name 'Circle' is appropriate as it is a noun and represents an entity.